City-of-Helsinki/helsinki-paatokset
# PP-254: Policymaker accessbility fixes

Accessibility improvements to policymaker pages.

To test: 
- Check out to branch. Run `drush cr`.

Check visual changes: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto
- Stream announcement block and color label are now within wrapped with `<div role="region">` with appropriate aria-labels
- Announcement block, in-page menu ("on this page you will find...") headings are now wrapped inside h2-elements
- Current page in side navigation has `aria-current="page"` -value

Documents-page:
- Year select input elements now have `aria-pressed` attribute.

Move to a minute page by clicking one from the documents list
- Minute page title is now wrapped in h1-element, including prefix and suffix ('minutes' and date after the title)
- Issue list -text is now a h2-element
- Title (HTML title element) now correlates with the page's h1-heading to make it more unique
